src/utils/DeploymentUtils.ts

```diff
@@ -1,5 +1,7 @@
-export {
+import * as contracts from "@across-protocol/contracts-v2";
```

Comment:
This will again break the FE bundle unfortunately...

Reply:
The problem is that the `@across-protocol/contracts-v2` package contains modules that don't work in a browser env. In the [main entry file](https://github.com/across-protocol/contracts-v2/blob/master/index.ts) of the package, we also export everything via `export * from "..."`. This results in the behavior that nothing can be tree-shaken as soon as you import something from the "default" path `@across-protocol/contracts-v2`. That on the other hand, resulted in browser-incompatible modules being bundled into the SDK. But because the SDK can also not be tree-shaken (due to tsdx's default behavior to put everything into one file), building the frontend bundle fails.

So we used the workaround to only import browser-compatible modules via something like `@across-protocol/contracts-v2/dist/<MODULE>`. This would omit the browser-incompatible imports from the SDK bundle.

Reply:
Reverted. Thanks for continuing to call this out @dohaki